src/main/java/school/lemon/changerequest/java/introduction/hw1/Task8.java

```diff
@@ -1,7 +1,14 @@
 package school.lemon.changerequest.java.introduction.hw1;
 
 public class Task8 {
-    public static int calculateSum(int N) {
-        return 0;
+    public static int calculateSum(int N)
+    {
+        int Sum=0;
+        for(int i=0;(N+i)<=2*N;i++)
```

Comment:
You can start not from 0, but from N and there will be no need to add do `N+i` on each step.